src/sql/finalize.sql

```diff
@@ -340,6 +340,12 @@ CREATE OPERATOR <=> (
 
 -- List of functions
 
+CREATE FUNCTION text2vec_openai(input TEXT, model TEXT) RETURNS vector
+STRICT LANGUAGE c AS 'MODULE_PATHNAME', '_vectors_ai_embedding_vector_wrapper';
+
+CREATE FUNCTION text2vec_openai_v3(input TEXT) RETURNS vector
```

Comment:
There is also `text-embedding-3-large` so `3small` here is better. Let's also add `3large` and `ada002`.

Reply:
It's a substitute of default value, we only need to write the most modern one, and other could let user to pick `text2vec_openai`.

If we encode all models, there would be a huge amount of functions in the future.